inspirehep/modules/workflows/utils/__init__.py

```diff
@@ -147,12 +149,26 @@ def _try_to_log(logfn, *args, **kwargs):
     return _decorator
 
 
+def get_uri(uri, outdir=None):
```

Comment:
Mh, from the name of the function I would expect that this function returns an URI, while instead it returns a file from a URI... what about `get_file_from_uri` ?

Reply:
It's not returning a file, but just a path. Better `retrieve_uri`